bitnami/redis/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 16.0.0
+version: 16.0.1
```

Comment:
Please revert bumping the version and ignore the CI errors complaining about the version. It's not required when you're only updating the README.md